.env.example

```
@@ -121,3 +121,4 @@ NUXT_PUBLIC_SUBGRAPH_URI_60808="https://api.goldsky.com/api/public/project_cm4ia
 NUXT_PUBLIC_SUBGRAPH_URI_80094="https://api.goldsky.com/api/public/project_cm4iagnemt1wp01xn4gh1agft/subgraphs/euler-simple-berachain/latest/gn"
 NUXT_PUBLIC_SUBGRAPH_URI_8453="https://api.goldsky.com/api/public/project_cm4iagnemt1wp01xn4gh1agft/subgraphs/euler-simple-base/latest/gn"
 NUXT_PUBLIC_SUBGRAPH_URI_9745="https://api.goldsky.com/api/public/project_cm4iagnemt1wp01xn4gh1agft/subgraphs/euler-simple-plasma/latest/gn"
+NUXT_PUBLIC_SUBGRAPH_URI_999="https://api.goldsky.com/api/public/project_cm4iagnemt1wp01xn4gh1agft/subgraphs/euler-simple-hyperevm/1.0.1/gn"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)